Prepatch/rom/patch.txt

```diff
@@ -76,4 +76,12 @@
 0x0071717D : 0x0001 : 75                                : EB 
 
 // Jump past the VP SelWatch button enabled checks
-0x00798108 : 0x0005 : E8 F3 01 E7 FF                    : EB 2D 90 90 90 
\ No newline at end of file
+0x00798108 : 0x0005 : E8 F3 01 E7 FF                    : EB 2D 90 90 90 
+
+// Disable jittering Temporal AA thingy (by @lybxlpsv)
+0x002E76D0 : 0x0004 : 8B 44 24 04                       : 29 C0 90 90
```

Comment:
Wouldn't it be plausible to make this a config option that toggles the flag at runtime?

Reply:
Good idea tho... fork me daddy...

Reply:
just comment it for now if you want to put it in patch.txt, unless if you want to create a component that disables mlaa, taa and enables toon shader if arbitrary resolution.

Reply:
Okay, I will revert back the `patch.txt` file to its original state.

Reply:
To add onto what was said here, I got the other hex edits that lybxlpsv made, inside of an optional field that could be commented out of the patch.txt file (if said user doesn't want it).

`// --- Optional Fixes: ---

// Disable Temporal AA (by @lybxlpsv)
0x002E76D0 : 0x0004 : 8B 44 24 04                       : 29 C0 90 90 

// Disable MLAA (by @lybxlpsv)
0x002EDAF8 : 0x0003 : 8B 45 10                          : 83 E0 00 

// Force Toon Shader (on PVs that use it) with arbitary resolutions (Higher than 720p)
0x00314F86 : 0x0002 : 74 0D                             : 90 90 `

Reply:
Just made a mistake that I forgot to correct before sending it, so I fixed it to reflect those changes.

Reply:
@Bryceless  You have to add 0x00400C00 onto those offsets for them to actually work; those values as-is do literally nothing

Reply:
OOF. I'll go and fix that

Reply:
I thought that the "0x0004" just determined the length of the offset, at least that's from what I've heard.

Reply:
Other lengths that have 4 hex values in the patch.txt use the same exact value for four different hex values that need to be modified, as an example.

Reply:
Alright. So after getting some assistance regarding how to get the correct hex address for patching, I managed to succeed in changing the addresses, so all of those fixes that I had on my patch.txt actually worked, alongside adding a segment where I can patch the exe to use mp4 files instead of wmvs.

`// --- Optional Fixes: ---

// Disable Temporal AA (by @lybxlpsv)
0x006E82D0 : 0x0004 : 8B 44 24 04                       : 29 C0 90 90 

// Disable MLAA (by @lybxlpsv)
0x006EE6F8 : 0x0003 : 8B 45 10                          : 83 E0 00 

// Force Toon Shader (on PVs that use it) with arbitary resolutions (Higher than 720p) (by @lybxlpsv)
0x00715B86 : 0x0002 : 74 0D                             : 90 90 

// Use MP4 files instead of WMV files (So PVs that rely on videos can be played back without issues) (By @Bryceless)
0x00B70069 : 0x0003 : 77 6D 76                          : 6D 70 34 
0x00B700B5 : 0x0003 : 77 6D 76                          : 6D 70 34 
0x00B7BA5A : 0x0003 : 77 6D 76                          : 6D 70 34 
0x00B8A5B7 : 0x0003 : 77 6D 76                          : 6D 70 34 
0x00B8A5D0 : 0x0003 : 77 6D 76                          : 6D 70 34 
0x00B8CE30 : 0x0003 : 77 6D 76                          : 6D 70 34 
0x00B8CE40 : 0x0003 : 77 6D 76                          : 6D 70 34 
0x00B8CEF0 : 0x0003 : 77 6D 76                          : 6D 70 34 
0x00B8CF00 : 0x0003 : 77 6D 76                          : 6D 70 34 
0x00B8CF20 : 0x0003 : 77 6D 76                          : 6D 70 34 
`

Reply:
Thanks for your contributions guys, you guys sure have more contribution and more IQ. You guys are definitely deserve having AFT more that @Filo97 kiddo. The sad thing is our goddess @samyuu don't want to merge those in his master just because those are not appropriate for average low IQ players. There are people can't even install ELAC. It's such a pity since they already has chance to have the PDA dump very publicized.

Reply:
You know what @rakisaionji i do agree there are quite a few people who deserve it more than me. That doesn't make you one of them.

Reply:
my dick is so hard rn